tests/framework/utils/k8s_helper.go

```diff
@@ -1376,16 +1376,20 @@ func (k8sh *K8sHelper) getPodsLogs(pods *v1.PodList, namespace, testName, platfo
 
 func (k8sh *K8sHelper) createTestLogFile(platformName, name, namespace, testName, suffix string) (*os.File, error) {
 	dir, _ := os.Getwd()
-	logDir := path.Join(dir, "_output/tests/")
+	baseLogDir := path.Join(dir, "_output/tests/")
+
+	// testName can contain a slach (e.g., `CephUpgradeSuite/TestUpgradeHelm`)
```

Comment:
```suggestion
	// testName can contain a slash (e.g., `CephUpgradeSuite/TestUpgradeHelm`)
```

Reply:
A simpler alternative might be to replace the slashes in `testName` with an underscore, but I'm fine with this as long as we can see that the files from subdirectories are gathered too.

Reply:
@travisn Yeah, that sounds simpler and safer for the current log collection.